frontend/src/scenes/notebooks/Notebook/Editor.tsx

```diff
@@ -202,3 +228,27 @@ export function lastChildOfType(node: Node, type: string, direct: boolean = true
     })
     return latestNode
 }
+
+export function useNotebookLink(href: string): { onClick: () => void } {
+    const { shortId } = useValues(notebookLogic)
+
+    const [path, search, internal] = useMemo(() => {
+        const url = new URL(href, window.location.origin)
+        const path = url.pathname
+        const search = url.search
+        const internal = window.location.origin === url.origin
+
+        return [path, search, internal]
+    }, [href])
+
+    const onClick = (): void => {
+        if (internal) {
+            router.actions.push(path + search)
+            openNotebook(shortId, NotebookTarget.Sidebar)
+        } else {
+            window.open(href, '_blank')
+        }
+    }
```

Comment:
I feel like you might be duplicating a lot of stuff that the `<Link>` component already does...
It already checks for internal versus external links and routes accordingly. The only addition needed here would be to check the `internal` part in the NotebookNodeLink and then pass set the property `targetBlank` on the `Link` component if it is external.

The big benefit of using the built in `Link` routing is you can do things like alt-click to open in a new page etc.